frontend/src/pages/Artwork/Artwork.jsx

```diff
@@ -11,27 +11,47 @@ import {
 import exampleArt from "../../assets/example-art.jpg";
 import MetricBadge from "../../components/MetricBadge/MetricBadge";
 import Tag from "../../components/Tag/Tag";
+import { useParams } from "react-router-dom";
+import { gql, useQuery } from '@apollo/client';
 
-const artwork = {
-  title: "Some Artwork",
-  description:
-    "Lorem ipsum dolor sit amet consectetur, adipisicing elit. Laboriosam est atque, similique dignissimos itaque ipsum, nemo, dolore modi tempore esse voluptatem temporibus? Recusandae molestias dolor non beatae nobis enim debitis!",
-  tags: ["very", "noice", "tag3"],
-  metrics: [{ value: 17, unit: "People Visited" },
+const metrics = [{ value: 17, unit: "People Visited" },
             { value: 12, unit: "Related Pieces"},
-            { value: 53, unit: "Comments"}],
-  rating: 3.2,
-};
+            { value: 53, unit: "Comments"}];
+const rating = 3.2;
 
 export default function Artwork() {
+  const { id } = useParams();
+  const query = gql`
+    query {
+      artwork (id:"${id}") {
+        edges {
+          node {
+            title
+            description
+            tags
+          }
+        }
+      }
+    }
+  `;
+
+  const { loading, error, data } = useQuery(query);
+
+  if (loading) return (<h1>Loading...</h1>);
+  if (error) return (<h1>Error: {error.message}</h1>);
```

Comment:
Sounds good. If I wanted to use the same error message class, should I copy that class name to the Artwork scss file, or would it be better to put it in index.scss and have it cascade to both the Profile and the Artwork pages?

Reply:
Or maybe we could even make the error message its own component, since I think I want to use it twice but pass it different text to display

Reply:
You could make a `<ConnectionErrorMessage/>` component, since you'll need JavaScript to trigger the "go back" functionality. You should just be able to lift the structure and styling into a component at `src/components/ConnectionErrorMessage` and import it to both locations. Lmk if you need help with this.

Reply:
You could also design your own. It doesn't need to look the same as mine if thats easier.

Reply:
Ok I'll try making the component

Reply:
For the goBack method, can I have the ConnectionErrorMessage component call useHistory() itself, or does useHistory() have to be called by the page component and passed to the ConnectionErrorMessage component to get the right history?

Reply:
Never mind figured that out I think, the only problem I'm having now is the scss doesn't seem to be properly applying... all I did was copy what was under the .error-message className and put it under a .ConnectionErrorMessage component in a fresh scss file. Did I miss something?

Reply:
You'll have to import that SCSS file into the new component

Reply:
Yeah I think I did that

Reply:
I just pushed the updates I've made, can you tell why the style isn't applying right?

Reply:
Your class name in HTML is `error-message`, but the class you are styling is `ConnectionErrorMessage`. These have to match

Reply:
Oh I see, it applies to the className, not the component name

Reply:
Ok I fixed it, lmk if anything else needs changing

Reply:
Actually hold on don't review it yet I still need to insert the ConnectionErrorMessage into the Profile page as well.